### ISSUE TITLE
Update the logo of open food facts in README.md

### ISSUE THREAD

**Issue by pratyaksh1610** [openfoodfacts/openfoodfacts-androidapp#4832]
The logo of open food facts is not up to date.

**Comment by VaiTon** [comment]
Yup, you're right! If you want to work on this I can assign you!

**Comment by teolemon** [comment]
The logos of secondary projects aren't either.
https://drive.google.com/drive/u/1/folders/1-0BEVMzkGQ1bRZ6XkdTnX5h74ilkKRFD
https://drive.google.com/drive/u/1/folders/1IgeI4jtLulSLhKiK1nIy8RAsqW7y8DYA

**Comment by teolemon** [comment]
We could update the logo in the app itself, and especially in its flavors

**Comment by pratyaksh1610** [comment]
I think we can extend and update the issue involving two tasks :  

- [ ] Update the logo in README.md
- [ ] Update the logos (including 4 flavors Open Food Facts, Open Beauty Facts, Open Pet Food Facts and Open Products Facts in the android app

**Comment by BhuvanAde** [comment]
As a UI/UX designer, I believe I can solve this issue as mentioned.

**Comment by teolemon** [comment]
- Related: https://github.com/openfoodfacts/openfoodfacts-androidapp/issues/4848
